src/Components/Authentication/Desktop/ResetPasswordForm.tsx

```diff
@@ -28,7 +36,7 @@ export const ResetPasswordForm: FormComponentType = props => {
         isValid,
       }: FormikProps<InputValues>) => {
         return (
-          <Form onSubmit={handleSubmit}>
+          <Form onSubmit={handleSubmit} height={180}>
```

Comment:
Shouldn't `height` be `maxHeight`? 

I thought we wanted to compute the height of the forms based on the children and animate height changes when errors show up. At least that's how I remember seeing it on Owen's prototype.

Reply:
Yeah, the tricky thing is we want to change the height of the form, but not of the modal itself. The form needs a fixed height that accommodates for the maximum number of errors. So the desired behavior looks like this:

![max-errors](https://user-images.githubusercontent.com/2236794/41257548-41eed99e-6d9b-11e8-8aa6-5980c09451b2.gif)

Reply:
Got it, what you did makes sense then. Thanks for the explanation!